main.cxx

```diff
@@ -0,0 +1,71 @@
+#include <algorithm>
+#include <iostream>
+#include <iterator>
+#include <string>
+#include <string_view>
+#include <vector>
+
+template<class Iterator>
+std::vector<int> scoresBetween(const Iterator &, const Iterator &, const Iterator &, const Iterator &);
+std::string lcs(const std::string_view &, const std::string_view &);
+
+int main() {
+	int T;
+	std::ios::sync_with_stdio(false);
+
+	std::cin >> T;
+	while (T--) {
+		std::string X, Y;
```

Comment:
How about declaring outside the while loop?

Reply:
No, the context of these should belongs to inside this loop.
If you're considering performance, compile will handle this. 😄